src/device.rs

```diff
@@ -16,9 +16,14 @@ pub use cl3::device::*;
 pub use cl3::ffi::cl_ext::{cl_amd_device_topology, CL_LUID_SIZE_KHR, CL_UUID_SIZE_KHR};
 
 use super::Result;
+#[cfg(feature = "CL_VERSION_1_2")]
```

Comment:
Please use `#[allow(unused_imports)]` instead of specifying specific types for versions.